CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-10-23, Version 2.2.0 (Beta), @dominickramer
+
+This release includes experimental support for the new [V8 Inspector Protocol](https://chromedevtools.github.io/debugger-protocol-viewer/v8/) ([#329](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/329)) and fixes an issue with setting snapshots on lines that cannot directly have snapshots set, such as comments ([#330](https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/pull/330)).
```

Comment:
Make this a bulleted list. The next two paragraphs have something to do with the first half of this sentence so it would be good to group them together onto the same bullet.

Reply:
Done